GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/PrefetchVerbTests.cs

```diff
@@ -145,6 +145,41 @@ public void PrefetchCleansUpPackDir()
             multiPackIndexLockFile.ShouldNotExistOnDisk(this.fileSystem);
         }
 
+        [TestCase, Order(12)]
+        public void PrefetchFilesFromFileListFile()
+        {
+            try
+            {
+                File.WriteAllLines(
+                    "fileList",
+                    new[]
+                    {
+                        Path.Combine("GVFS", "GVFS", "Program.cs"),
+                        Path.Combine("GVFS", "GVFS.FunctionalTests", "GVFS.FunctionalTests.csproj")
+                    });
+
+                this.ExpectBlobCount(this.Enlistment.Prefetch("--files-list fileList"), 2);
+            }
+            finally
+            {
+                File.Delete("fileList");
+            }
+        }
+
+        [TestCase, Order(13)]
+        public void PrefetchFilesFromFileListStdIn()
+        {
+            var input = string.Join(
+                "\n",
+                new[]
+                {
+                    Path.Combine("GVFS", "GVFS", "packages.config"),
+                    Path.Combine("GVFS", "GVFS.FunctionalTests", "App.config")
+                });
+
+            this.ExpectBlobCount(this.Enlistment.Prefetch("--files-list stdin", standardInput: input), 2);

```

Comment:
If someone had a file named "stdin", this would break the existing behavior. Yes, unlikely, but easily fixed by creating new arguments, say `--stdin-files` and `--stdin-folders`.

Reply:
(Sorry, I see that you are adding `--files-list`, so there is no existing behavior to break. However, doing the same pattern in `--folders-list` would break existing behavior, and it would be best to stay consistent.)